docs/lob_guide/src/asciidoc/_chapters/about.adoc

```diff
@@ -0,0 +1,195 @@
+////
+/**
+* @@@ START COPYRIGHT @@@
+*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+* @@@ END COPYRIGHT @@@
+*/
+////
+
+[[About_This_Document]]
+= About This Document
+This guide dwscribes how to use Large Object (LOB) datatypes.
+
+[[Intended_Audience]]
+== Intended Audience
+This manual is intended for programmers who use LOB datatypes. 
+
+[[New_and_Changed_Information]]
+== New and Changed Information
+This is a new guide.
+
+[[Document_Organization]]
+== Document Organization
+
+[cols="30%,70%",options="header"]
+|===
+|Chapter                                       | Description
+| <<Introduction,Introduction>>                | Introduces LOB and covers following topics: +
+&#8226; 1.1 <<what is lob,What is LOB>> +

```

Comment:
Grammar: "what is a lob,What is a LOB" is better.

Reply:
Thanks Dave.
I've corrected. :)